index.bs

```diff
@@ -3676,85 +3690,85 @@ the same procedure as other [=assertion signatures=] generated by the [=authenti
-        a multi-device credential. 
+        When `BE` [=flag=] is set to `0`, the [=generating authenticator=] will never allow the credential to transition from a 
+        [=single-device credential=] to a [=multi-device credential=]. 
         
-        A single-device credential is not durable and cannot survive single device loss. [=Relying Party|Relying Parties=]
-        should ensure that an account has additional [=authenticators=] enrolled and/or an account recovery process in place.
+        A [=single-device credential=] is not durable and cannot survive single device loss. [=[RPS]=] should ensure that a [=user account=] 
+        has additional [=authenticators=] [=registration ceremony|registered=] and/or an account recovery process in place.
 
-        For example, the user could be prompted to set up a single-device credential on a [=roaming authenticator=], like a 
-        security key, or another [=authenticator=] that is capable of holding multi-device credentials.
+        For example, the user could be prompted to set up an additional [=authenticator=], such as a [=roaming authenticator=] or an 
+        [=authenticator=] that is capable of [=multi-device credentials=].
     
     - Upgrading a user to a password-free account:
 
-        When bit 4 changes from `0` to `1`, the [=authenticator=] is signaling that the [=Public Key Credential Source|credential source=]
-        is durable (it has been backed up and is protected from single device loss). This is often referred to as a multi-device credential.
+        When the `BS` [=flag=] changes from `0` to `1`, the [=authenticator=] is signaling that the [=credential=] is backed up and is protected from single device loss. 
+        This is often referred to as a [=multi-device credential=]
         
         A [=Relying Party=] may decide to prompt the user to upgrade their account security and remove their password.
     
     - Adding an additional factor after a state change:
 
-        When bit 4 changes from `1` to `0`, the [=authenticator=] is signaling that the [=Public Key Credential Source|credential source=]
-        is no longer durable (not backed up and is not protected from single device loss). This could be the result of the user deleting the 
-        credential, deleting a backup account, an issue with the backup service, or another undefined error condition.
+        When the `BS` [=flag=] changes from `1` to `0`, the [=authenticator=] is signaling that the [=credential=] is no longer backed up, 
+        and no longer protected from single device loss. This could be the result of the user actions, such as disabling the backup service, 
+        or errors, such as issues with the backup service.
 
         When this transition occurs, the credential is no longer durable and a [=Relying Party=] should guide the user through a process to 
```

Comment:
Same thing about "durable", and we've already stated that the credential is no longer backed up.

```suggestion
        When this transition occurs, the [=Relying Party=] should guide the user through a process to 
```